src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/WebJobs.Logging.ApplicationInsights.csproj

```diff
@@ -27,7 +27,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Identity" Version="1.10.4" />
+    <PackageReference Include="Azure.Identity" Version="1.10.2" />
```

Comment:
This shouldn't be getting rolled back?

Reply:
I am aligning this with what we have in the host https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/WebJobs.Script.csproj#L43

Reply:
I think this was explicitly moved forward a couple weeks ago -- https://github.com/Azure/azure-webjobs-sdk/pull/3046/files#diff-97ff29e7c186ba87bf63628248cdec6f393d6c38410d71f3f68232554c126db6.

When we move this to the host, we'll also move that forward.

Reply:
Done.